src/Presentation.js

```diff
@@ -76,11 +76,11 @@ export default class Presentation extends React.Component {
             </Heading>
           </Slide>
           <Slide transition={["zoom", "fade"]} bgColor="primary" notes="<ul><li>talk about that</li><li>and that</li></ul>">
-            <CodePane
+            {/*<CodePane
               lang="jsx"
               source={require("raw!../assets/deck.example")}
               margin="20px auto"
-            />
```

Comment:
Disabled, because there is no `raw-loader` in CRA.

@gaearon here is a real example of the issue that I have mentioned [there](https://mobile.twitter.com/boriscoder/status/777093281662562304).
So far I don't know how to receive content of text file synchronously with CRA.